docs/mesos/DCOS.md

```diff
@@ -51,6 +51,15 @@ You can check the status of etcd by visiting the **Services**
 tab. You can install Calico once etcd appears with a status
 of **`Healthy`** (note that this could take a few minutes).
 
+### Mesos-DNS
+Calico is compatible with Mesos DNS, but requires a slight configuration change to prefer Container IP addresses over the host's. Follow these steps to allow containers to be resolvable via their containerized IP: 
```

Comment:
Personal preference:  slight->minor   (not sure why slight sounds odd to me)

Maybe "to use the containers IP address rather than the hosts".